tests/features/project.feature

```diff
@@ -7,17 +7,26 @@ Feature: Project content creation
   @javascript
   Scenario: Creation of a Project content through the UI.
     Given I am logged in as a user with the "create oe_project content, access content, edit own oe_project content, view published skos concept entities, manage corporate content entities" permission
+    # Create Stakeholders.
+    And the following Stakeholder Organisation entity:
+      | Name       | Coordinators stakeholder |
+      | Published  | Yes                      |
+    And the following Stakeholder Organisation entity:
+      | Name      | Participants stakeholder  |
+      | Published | Yes                       |
```

Comment:
Let's use here all fields that are parsed in `alterStakeholderOrganisationFields` so that we also test that they work properly.

Reply:
done